src/Symfony/Bundle/FrameworkBundle/Test/MessengerAssertionsTrait.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Test;
+
+use Symfony\Component\Messenger\EventListener\MessagesSentToTransportsListener;
+use Symfony\Component\Messenger\Test\Constraint as MessengerConstraint;
+
+/*
+ * @author Marilena Ruffelaere <marilena.ruffelaere@gmail.com>
+ */
+
+trait MessengerAssertionsTrait
+{
+    /**
+     * @param int         $count       The expected number of messages
+     * @param string|null $busName     The busName to consider
+     * @param string|null $messageFQCN The message object class name to consider
+     */
+    public static function assertMessagesCount(int $count, string $busName = null, string $messageFQCN = null, string $message = ''): void
```

Comment:
Very small suggestion : I think `assetMessageCount` (without the 's') would be more 'correct' :smiley: